lib/src/main/java/tech/relaycorp/relaydroid/endpoint/FirstPartyEndpoint.kt

```diff
@@ -29,13 +29,19 @@ internal constructor(
     public fun issueAuthorization(
         privateThirdPartyPublicKey: PublicKey,
         expiryDate: ZonedDateTime
-    ): Certificate =
-        issueDeliveryAuthorization(
+    ): AuthorizationBundle {
+        val pda = issueDeliveryAuthorization(
             subjectPublicKey = privateThirdPartyPublicKey,
             issuerPrivateKey = keyPair.private,
             validityEndDate = expiryDate,
-            issuerCertificate = certificate
+            issuerCertificate = identityCertificate
         )
+        val pdaChain = listOf(identityCertificate, gatewayCertificate)
+        return AuthorizationBundle(
+            pda.serialize(),
+            pdaChain.map { it.serialize() }
```

Comment:
I'm not super happy about doing the serialisation inside this method, but the alternative would be to do it inside `AuthorizationBundle`, which would be problematic:

- Its constructor would have to take `Certificates`, which would require making the constructor `internal`.
- If the constructor is `internal`, it'd be a pain in the neck for people to initialise this class in their tests.

Reply:
I think it's fine for it to be here.